docker-compose.yml

```diff
@@ -104,8 +108,19 @@ services:
       - "8000:8000"
       - "3000:3000"
       - "6006:6006"
+  watchtower:
+    image: containrrr/watchtower
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    command: --http-api-update --label-enable
+    environment:
+      - WATCHTOWER_HTTP_API_TOKEN=mytoken
```

Comment:
We'll change this, correct? Or is this a variable that we feed an actual token into otherwise?

If it needs to be modified by the user, we need to extract this to the `.env` file.

Reply:
Watchtower's HTTP API will only be accessible by the service container; won't be accessible to the frontend. I was thinking we leave it as is. I can see this being an attack vector, but it's a moon shot. What do you think?

Reply:
Maybe useful illustration: https://github.com/centerofci/mathesar/pull/2406/files#diff-dc39d421dba814d90579e0f66a950688aa9efc75c67f967b33858ffc9ef24ec3R29

Reply:
> Watchtower's HTTP API will only be accessible by the service container; won't be accessible to the frontend. I was thinking we leave it as is. I can see this being an attack vector, but it's a moon shot. What do you think?

Fair enough, good point.